libs/agno/agno/os/schema.py

```diff
@@ -862,6 +869,12 @@ def from_dict(cls, run_dict: Dict[str, Any]) -> "RunSchema":
             events=[event for event in run_dict["events"]] if run_dict.get("events") else None,
             references=run_dict.get("references", []),
             reasoning_messages=run_dict.get("reasoning_messages", []),
+            images=extract_media(run_dict, "images"),
+            videos=extract_media(run_dict, "videos"),
+            audio=extract_media(run_dict, "audio"),
+            audio_output=extract_media(run_dict, "audio_output"),
```

Comment:
no i dont think we need audio_output over here. just response_audio and audio.
audio_output is stored internally in model response